src/urlhandlers/mock_node_url_handler.js

```diff
@@ -0,0 +1,7 @@
+export class NodeURLHandler {
```

Comment:
I'd add a remark in `url_handler` above this line

     import { XHRURLHandler } from './urlhandlers/xhr_url_handler';

telling that for a browser build, this will be replaced to a mock at build-time, so that the people reading the code will understand it